07-HTML5-APIS/Storage and WebService/script.js

```diff
@@ -0,0 +1,44 @@
+
+
+function save(){
+	localStorage.setItem("item", $("#input1").val());
```

Comment:
A good practice, is caching jQuery selectors, if you are using the same one more than once.
var $input1 =  $('#input1');
....
$input1.val()